cnf/ext/central.maven

```diff
@@ -0,0 +1,4 @@
+# List repository contents using GAV coordinates
+org.bndtools:org.bndtools.templates.osgi:5.2.0
```

Comment:
The versions should use a macro so that it can be centrally edited.

Reply:
Good point. Do we actually something like a prepared macro like current_bndversion?

Reply:
Seems like it might be a nice macro to add.

Reply:
https://github.com/bndtools/bnd/issues/4485